src/app/shared/styles/elements/utils.scss

```diff
@@ -49,10 +49,6 @@
     }
 }
 
-.icon {
```

Comment:
Class is used in multiple places in argo-cd-ui: https://github.com/argoproj/argo-cd-ui/search?q=className%3D%27icon&unscoped_q=className%3D%27icon


Did you replace it with something else?